en/docs/Learn/APIMonetization/monetizing-an-api.md

```diff
@@ -0,0 +1,703 @@
+    ```
+
+    -   `<monetization-usage-publish-token>` - This is the token that you got when you executed the Admin REST API with the monetization usage scope.
+
+    The REST API response is as follows:
+
+    ``` java
+    {"status":"Request Accepted","message":"Server is running the usage publisher"}
+    ```
+
+    The aggregation count will be stored in `ApiPerDestinationAgg_SECONDS` table, which corresponds to the WSO2 API-M Analytics database. If you check the `AM_MONETIZATION_USAGE_PUBLISHER` table after the admin API call is made, you will see that the record for the API usage is published.  
+      
+    In addition, after the admin API call is made the aggregated bill
+    gets generated in Stripe (connected account).  
+    ![Pricing](../../assets/img/Learn/pricing.png) At the end of
+    the billing cycle, the amount will be charged from the customer. As
+    this scenario uses a usage based billing model, the revenue goes from the subscriber to the API provider in the billing engine.
+
+#### Step 5 - Monitor usage of a monetized API
+
+There are two types of billing plans for monetized APIs, namely the fixed billing plan and the dynamic billing plan. Usage needs to be monitored for APIs associated with dynamic billing plans, which is also referred to as metered billing. 
+
+When an API request is initiated, the API Gateway will publish the analytics related to that request. The successful API calls is recorded in the database and this data is used to aggregate the API usage and charge from the subscriber.
+
+##### View the revenue for a particular subscription in a given API
+
+If a subscription is based on metered billing it is possible to get the revenue up-to now, which is within the current billing cycle. Subscriptions with a fixed amount is not counted here.
+
+##### View the revenue for a given API from all subscriptions
+
+This refers to the revenue related to the aggregation of all metered billing subscriptions. The API provider can use this chart to get an idea of the total revenue up-to now for each subscription.
+
+### Disable monetization
+
+Follow the instructions below to disable monetization for an API:
+
+<html>
+      <div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>It is assumed that you have already [configured the additional properties](../../Learn/APIMonetization/monetizing-an-api#additionalProp) that are specific to the billing engine with regard to monetization in WSO2 API Manager.</p>
+      </div> 
+</html>
+
+1.  Sign in to the WSO2 API Manager Publisher.  
+    `https://<hostname>:9443/publisher-new`
+2.  Click on the monetized API.
+3.  Click **Monetization** to go to the Monetization configurations.  
+    ![Disable monetization](../../assets/img/Learn/disable-monetization.png)
+4.  Click **Enable Monetization** to unselect the enable monetization option.
```

Comment:
is this a checkbox?

Reply:
According to the Microsoft Manual of Style we do not explicitly state whether it is a checkbox or a button etc. In this case too I have not specified same. However, as there is a screenshot it shows that it is a checkbox.